### ISSUE TITLE
Backdrop Filters don't appear initially in live demo

### ISSUE THREAD

**Issue by Jinsung-L** [Jinsung-L/tailwindcss-filters#1]
Browser: Safari Version 14.0 (15610.1.28.1.9, 15610)

This bug is very strange. The bug only appears on the production build of the demo page browsing with Safari.

I've also tested this on the other Next.js project that was created with `create-next-app`. But it didn't appear on there.

I confirmed that the all classes and related CSS codes are properly generated and applied. If you open up the Developer Tool and inspect the element, you will see that everything looks fine.

What is so weird is that the backdrop filter that wan't initially appeared suddenly works only after there's a css class update.

In the Dev tool try either:

1. Update class attributes.
2. Toggle applied css styles.

Then it suddenly works!

**Comment by Jinsung-L** [comment]
It's somehow related to having fixed size for the parent of the overlay div.
Either adding width and height classes on the relatively positioned div, or adding `fixed` class on the overlay resolves the issue.

Still have no idea on why this bug is occurring or why it only appears on the production build but not in the dev mode.

There are few things to check.

- [x] Does this bug also appears on `create-react-app` project?
- [x] Is there any workaround?

**Comment by Jinsung-L** [comment]
Confirmed that the bug also appears on a project created with `create-react-app`. The bug even appears on the dev mode but still works fine in Chrome.

I could workaround this issue by forcing the parent to have fixed aspect ratio using [this plugin](https://github.com/tailwindlabs/tailwindcss-aspect-ratio). This seems to somehow makes Safari to be able to correctly calculate the size of the backdrop overlay.

I also confirmed that the bug appears on plain HTML and CSS codes. So, I think this a bug of Safari. The fact that the background color property of the overlay is properly applied but only the area where backdrop filter should apply couldn't be calculated without specific size information makes it more seems to be a bug.

https://codepen.io/jinsung-l/pen/ZEpGXed
In Editor View, it occasionally works. I have no idea on why.
